rotkehlchen/chain/evm/decoding/compound/v3/balances.py

```diff
@@ -169,8 +170,16 @@ def query_collateral(self, balances: BalancesSheetType) -> 'BalancesSheetType':
                     ),
                 ))
 
+        chunk_size, call_order = get_chunk_size_call_order(
+            evm_inquirer=self.evm_inquirer,
+            arbiscan_chunksize=ARBITRUM_ETHERSCAN_MULTICALL_CHUNK_SIZE,
+        )
```

Comment:
you calculate it twice. Can you make query_liabilies and query collateral take it as argument and then calculate it once in `query_balances`